api-spec/payment-requests-api.yaml

```diff
@@ -165,6 +167,11 @@ paths:
               schema:
                 $ref: '#/components/schemas/ProblemJson'
 components:
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: X-Api-Key
```

Comment:
As a best practice ( for me:-) ), it's better to use all lowercase since headers are case-insensitive (RFC 7230).